obspy/clients/filesystem/tests/test_sds.py

```diff
@@ -222,6 +222,48 @@ def test_sds_report(self):
             if failed:
                 raise Exception
 
+    def test_get_waveforms_bulk(self):
+        """
+        Test get_waveforms_bulk method.
+        """
+        for year, doy in ((2015, 213), (2015, 1), (2012, 1)):
+            t = UTCDateTime("%d-%03dT00:00:00" % (year, doy))
+            with TemporarySDSDirectory(year=year, doy=doy) as temp_sds:
+                chunks = [
+                    ["AB", "XYZ", "", "HHZ", t,
+                    t + 20, os.path.join(temp_sds.tempdir, "file_1.mseed")],
```

Comment:
please get rid of the filename/path input here. `get_waveforms_bulk` should only expect
 - network
 - station
 - location
 - channel
 - starttime
 - endtime
as argument list.